Cloud propagator package is missing one file and breaks compilation

Please answer these questions before submitting a bug report.

### What version of OpenTelemetry are you using?

0.10.1

### What version of Node are you using?

13.12.0

### What did you do?
If possible, provide a recipe for reproducing the error.

used cloud trace propagator as per instructions at https://www.npmjs.com/package/@google-cloud/opentelemetry-cloud-trace-propagator

### What did you expect to see?

The trace propagator working without errors.

### What did you see instead?

```
Error: Cannot find module './CloudPropagator'
Require stack:
- .../node_modules/@google-cloud/opentelemetry-cloud-trace-propagator/build/src/index.js:26:14
```

### Additional context
Add any other context about the problem here.

The CloudPropagator.ts file added in #116 does not seem to be included in the published NPM module.

--- a/package.json
+++ b/package.json
@@ -1,5 +1,6 @@
 {
   "name": "opentelemetry-operations-js",
+  "version": "0.4.0",
   "description": "OpenTelemetry is a distributed tracing and stats collection framework.",
   "scripts": {
     "clean": "lerna run clean",

--- a/packages/opentelemetry-cloud-trace-propagator/package.json
+++ b/packages/opentelemetry-cloud-trace-propagator/package.json
@@ -26,8 +26,8 @@
     "node": ">=8.0.0"
   },
   "files": [
-    "build/src/**/*/.js",
-    "build/src/**/*/.d.ts",
+    "build/src/**/*.js",
+    "build/src/**/*.d.ts",
     "doc",
     "LICENSE",
     "README.md"

